packages/gestalt/src/Video/Video.js

```diff
@@ -0,0 +1,273 @@
+// @flow
+
+import * as React from 'react';
+import * as fullscreen from './fullscreen';
```

Comment:
@pfarejowicz since we're only importing `fullscreen` and `VideoControls` in this file, why did we extract it out? Would love to hear what @chrislloyd thinks